client/post-editor/media-modal/index.jsx

```diff
@@ -209,15 +209,15 @@ module.exports = React.createClass( {
 		this.setView( ModalViews.IMAGE_EDITOR );
 	},
 
-	onImageEditorClose: function() {
+	onImageEditorClose: function( event, { hasEditedImage } = { hasEditedImage: false } ) {
```

Comment:
I pass this in the onDone handler, https://github.com/Automattic/wp-calypso/pull/7374/files#diff-1f462f2e28f6f7a1f08bd085a3fca110R68